02-express-tutorial/app.js

```diff
@@ -1 +1,29 @@
-console.log('Express Tutorial')
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const peopleRouter = require('./routes/people');
+
+app.use(express.static("./public"));
+app.use(express.static('./methods-public'));
+
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
As a note for future apps, remember you'll want this to be `express.urlencoded` and `express.json`